client/src/App.js

```diff
@@ -1,10 +1,44 @@
 import React from 'react';
+import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
+import allComponents from './Routes/allComponents';
+
+const {
+  LandingPage,
+  Login,
+  Signup,
+  Home,
+  Header,
+  Footer,
+  MyApplications,
+  MyOffers,
+  NewOffer,
+  OfferDetails,
+  SavedOffers,
+  Profile,
+  PageNotFound,
+} = allComponents;
 
 function App() {
   return (
-    <div>
-      <h1>React App</h1>
-    </div>
+    <Router>
+      <>
+        <Header />
+        <Switch>
+          <Route exact path="/" Component={LandingPage} />
```

Comment:
I think you should differentiate between public routes and private routes.
my suggestion is to use for example /app route for private routes, so every private route starts with /app.
my goal is to collect all private route on one component that contains all private routes, so App.js will be much readable and small.
thanks